packages/block-library/src/social-link/README.md

```diff
@@ -0,0 +1,21 @@
+# Social Icon block
+
+The Social Icon is a foundational block that displays various icons linking to different social profiles or sites. Each social service is registered as a variation of this block.
+
+
+## Adding new social icons
+
+The core maintains certain standards for social icons. To add a new one to the existing variations, it must be a well-established, popular, and mainstream service.
```

Comment:
"The core maintains certain standards for social icons."
Sounds a bit unclear
Maybe this is clearer: "WordPress features a default set of social icons."
Then also: 
"To add a new variation to WordPress, the social service must be well-established and popular."

Reply:
Maybe something like this:

> The core maintains certain standards for adding new social icons.

The "standards" refer to the guidelines themselves.